source/cpu/cpu.go

```diff
@@ -137,14 +132,15 @@ func (s Source) Discover() (source.Features, error) {
 
 // Check if any (online) CPUs have thread siblings
 func haveThreadSiblings() (bool, error) {
-	files, err := ioutil.ReadDir(cpuDevicesBaseDir)
+
+	files, err := ioutil.ReadDir(source.SysfsDir.Path("cpu/devices"))
```

Comment:
Previously we were reading `/host-sys/bus/cpu/devices` and now we are reading `/host-sys/cpu/devices` which does not exist AFAIK. I think we are missing the `bus` part here

Reply:
Argh 🤯 This bug obviously creeped in the latest version of my patch where I introduced the `HostDir` type, and, I must've tested the previous oudated version 😭 Thanks for catching this 🎏 Fixed now